python/cuml/cuml/internals/mixins.py

```diff
@@ -45,6 +44,34 @@
 }
 
 
+class _tags_class_and_instance:
```

Comment:
This was moved out of `base_helpers.py` since that file was removed (and it's only used in this file).